src/main/scala/org/apache/spark/shuffle/crail/CrailShuffleManager.scala

```diff
@@ -91,19 +88,36 @@ private[spark] class CrailShuffleManager(conf: SparkConf) extends ShuffleManager
     logInfo("shutting down crail shuffle manager")
     CrailDispatcher.put
   }
+
+
+
+  override def getReaderForRange[K, C](
```

Comment:
Unclear how to specify a range for CrailShuffleReader.

Reply:
Can you clarify?